src/Sulu/Bundle/LocationBundle/Resources/config/geolocator.xml

```diff
@@ -9,19 +9,22 @@
 
     <services>
         <!-- Guzzle client -->
-        <service id="sulu_location.geolocator.guzzle.client" class="%sulu_location.guzzle.client.class%"/>
+        <service id="sulu_location.geolocator.guzzle.client" alias="sulu_location.geolocator.guzzle.client_deprecated">
+            <deprecated />
+        </service>
+        <service id="sulu_location.geolocator.guzzle.client_deprecated" class="%sulu_location.guzzle.client.class%"/>
```

Comment:
Do we need this service definition can we not just set the class on the above service definition which has the `<deprecated />` or can only aliases be deprecated?

Reply:
Happy new year. O yes your right i can also depricate services. I read the doc's wrong i think 
https://symfony.com/blog/new-in-symfony-4-3-deprecating-service-aliases

but this is been here for services all along https://symfony.com/blog/new-in-symfony-2-8-deprecated-service-definitions 

So nice to learn all this. Thank you for your time in helping me with this.

Reply:
lol wou i should have checked that better. :D

Reply:
Thx also a Happy new year!. Good to hear that you also learn from this PR :).